src/marine/ndbc_hfradar2ioda.py

```diff
@@ -117,21 +115,23 @@ def main():
     args = parser.parse_args()
     fdate = datetime.strptime(args.date, '%Y%m%d%H')
     VarDims = {
-        'sea_water_meridional_current': ['nlocs'],
-        'sea_water_zonal_current': ['nlocs']}
+        'vComponentOfCurrent': ['Location'],
```

Comment:
I am not very fond of these two variable names.  I had already renamed these as:
```
  waterZonalVelocity
  waterMeridionalVelocity
```
as part of my ioda PR 673.  Would you mind changing these to match please and then make your PR depend on 673 in ioda and we could always change our minds later on these variables if someone in ocean group has a problem?

Reply:
Changed as suggested. Re-tested and committed.